### PR TITLE
Add text to regulations and standards

### PR DESCRIPTION
**Pull request by jhdalek55**
This is an extremely rough draft. I had a surprisingly difficult time coming up with short descriptions of these regulations. The official pages for the two ISO standards literally list only what iteration they were on and title.  I could find nothing in the way of a general explanation of ISO21434. I also know that there are probably a few more regulations that should be on this initial post. I just wanted to get something on the page so we can begin to get this built. So, please, add, subtract, change or point me in the right direction for better sources. Thanks.

### CODE REVIEW

**Comment by jhdalek55** [comment]
@iramcdonald can you take a look and tell me what else needs to be added. Other comments also welcome.

**Comment by iramcdonald** [comment]
Hi Lois,

This all looks good to me.

I would recommend adding the following links (with project name labels):

ISO/AWI PAS 5112 Road vehicles — Guidelines for auditing cybersecurity
engineering (Work-in-Progress)
- https://www.iso.org/standard/80840.html?browse=tc
- This document will provide auditing guidelines for ISO/SAE 21434

ISO/SAE DIS 21434 Road vehicles — Cybersecurity engineering (February 2020)
- https://www.iso.org/standard/70918.html?browse=tc
- This document is the successor to SAE 3061

ISO/CD 24089 Road vehicles — Software update engineering (February 2021)
- https://www.iso.org/standard/77796.html?browse=tc
- This document augments ISO 26262 (Functional Safety) and ISO/SAE 21434
(Cybersecurity)
- This document standardizes detailed process requirements for wired and
wireless software update

ISO/DIS 21448 Road vehicles — Safety of the intended functionality (SOTIF)
(January 2021)
https://www.iso.org/standard/77490.html?browse=tc
- This document augments ISO 26262 (Functional Safety)

SAE 3061 Cybersecurity Guidebook for Cyber-Physical Vehicle Systems
(January 2016)
- https://www.sae.org/standards/content/j3061_201601/
- This document was the primary input to the ISO/SAE 21434 joint project

SAE 3101 Hardware Protected Security for Ground Vehicles (February 2020)
- https://www.sae.org/standards/content/j3101_202002/
- This document defines requirements for a hardware protected security
environment
- This document abstracts security components such as TCG TPM, GP SE, GP
TEE, HSM, etc.

Cheers,
- Ira

*Ira McDonald (Musician / Software Architect)*

*Chair - SAE Trust Anchors and Authentication TF*
*Co-Chair - TCG Trusted Mobility Solutions WG*

*Co-Chair - TCG Metadata Access Protocol SG*








*Chair - Linux Foundation Open Printing WGSecretary - IEEE-ISTO Printer
Working GroupCo-Chair - IEEE-ISTO PWG Internet Printing Protocol WGIETF
Designated Expert - IPP & Printer MIBBlue Roof Music / High North
Inchttp://sites.google.com/site/blueroofmusic
<http://sites.google.com/site/blueroofmusic>http://sites.google.com/site/highnorthinc
<http://sites.google.com/site/highnorthinc>mailto: blueroofmusic@gmail.com
<blueroofmusic@gmail.com>(permanent) PO Box 221  Grand Marais, MI 49839
906-494-2434*

**Comment by jhdalek55** [comment]
Thanks, Ira. I will add these.

Lois

File: regulations_and_standards.md [hunk 1]
```diff
@@ -3,4 +3,47 @@ layout: default
 css_id: regulations_and_standards
 ---
 
-# Related Regulations and Standards
+# Regulations and Standards Relevant to Uptane
+
+This page aims to provide the Uptane community with up-to-date information on the status of regulations and standards that may directly affect the evolution of the Uptane framework. By keeping abreast of these changes, mandated by both government agencies and industry best practices, we can ensure that Uptane will help the automtive industry achieve compliance, and support continuing improvement of cybersecurity defenses.
```
Comment (mnm678) [inline regulations_and_standards.md:8]:
```suggestion
This page aims to provide the Uptane community with up-to-date information on the status of regulations and standards that may directly affect the evolution of the Uptane Standard. By keeping abreast of these changes, mandated by both government agencies and industry best practices, we can ensure that Uptane will help the automtive industry achieve compliance, and support continuing improvement of cybersecurity defenses.
```

**Review by mnm678 — APPROVED** [review]
One minor comment, but otherwise LTGM

**Pull request merged**